ternary/ternary_axes_subplot.py

```diff
@@ -313,10 +313,10 @@ def legend(self, *args, **kwargs):
         ax = self.get_axes()
         ax.legend(*args, **kwargs)
 
-    def savefig(self, filename, dpi=200, format=None):
+    def savefig(self, filename, dpi=200, format=None, **kwargs):
```

Comment:
Can you remove dpi and format here and below? Since they are kwargs they'll get caught by `**kwargs` now as well

Reply:
Removing the `dpi` default of 200 would reduce the default resolution to the matplotlib default which is think is 100 (depending on your matplotlibrc file). 

Is there a worry that this will might cause a change in the behaviour of this `savefig` for people who have been relying on the default `dpi`.

Reply:
We can do something like this to maintain the default:
```
if 'dpi' not in kwargs:
    kwargs['dpi'] = 200
```